apps/prairielearn/src/pages/instructorInstanceAdminLti/instructorInstanceAdminLti.html.ts

```diff
@@ -140,27 +140,36 @@ function LtiCredentialsCard({
                   <tr>
                     <td>
                       <code>${config.ltiRedirectUrl}</code>
-                      <i
-                        class="far fa-copy"
-                        title="Copy to clipboard"
+                      <button
+                        type="button"
+                        class="btn btn-sm btn-ghost"
+                        aria-label="Copy redirect URL to clipboard"
```

Comment:
Minor accessibility improvement. LTI 1.1 isn't maintained but this will help us if it comes up on any future accessibility reviews.